### DISCUSSION TITLE
The direction this project is heading in

### DISCUSSION THREAD

**Discussion by TechStudent10** [collab-community/journey-book#46] [General]
Hello everyone working on Journey Book. I've been notified multiple times now that activity has boosted again. If you remember, I had some suggestions a couple of months ago. However, I see you haven't used those suggestions and I fear that the project is heading in a wrong direction in terms of implementation.

Right now, in order to create, update, or view a journey, you need to be on GitHub. This decision confuses me as not all developers are on GitHub. A lot of developers might be on other sites such as GitLab. You are basically limiting the usage of the project to GitHub users, which don't get me wrong, is a massive portion of the developer community. However, a lot of people don't want to use services from a big tech company such as Microsoft due to their anti-OSS attitude (not is much as Apple though). I am surprised you didn't go for the website option. It would make users' lives easier as they don't need to login to GitHub. The GitHub option also has a few risks. What if someone submits a PR that might include their journey, but also might've modified someone else's journey. I expect these PRs are reviewed, but there is always room for human error. The GitHub approach allows others to modify other users' journey, something I think no one wants.

In the project's current state, I won't be adding a journey. If you do want to convert this to a website, I am always willing to help.

Here is a summary of what needs to be done if you decide to go with this approach:
- Database creation
- Website UI and backend
- Security

This may sound a bit more complicated than the current approach, however I think it would help in the long run.

Edit: It has come to my realization that there is a website. However, you cannot create, view or update journeys. So it doesn't count.

-TS10

**Comment by KendallDoesCoding** [comment]
Hey @TechStudent10,

I don't think any of us are this experienced to make a site like this, but
maybe we'll do it in the future once we learn how to do it.

**Reply by TechStudent10** [reply]
"experienced enough"

Are you sure about that? Anyone can learn how to.

Well it's your project so it's your decision I guess.

**Reply by Adarsh-jaiss** [reply]
I don't want to be a site right now.. We will see later how it goes

**Reply by Adarsh-jaiss** [reply]
My primary focus is to give beginner's a platform to contribute to open source even though they don't know how to code

**Reply by KendallDoesCoding** [reply]
> My primary focus is to give beginner's a platform to contribute to open source even though they don't know how to code

👍
